.node-version

```diff
@@ -1 +1 @@
-18
\ No newline at end of file
+20
```

Comment:
Has no effect on our compiled code, this is just used for the local dev env of this package.